chia/server/rate_limits.py

```diff
@@ -116,7 +133,7 @@ def process_msg_and_check(self, message: Message) -> bool:
         Returns True if message can be processed successfully, false if a rate limit is passed.
         """
 
-        current_minute = time.time() // self.reset_seconds
+        current_minute = int(time.time() // self.reset_seconds)
```

Comment:
for some reason, integer division returns a float (according to mypy)

Reply:
It's because `//` is really `floor( div (x) )`

Toss a cast to int in there